auditbeat/module/file_integrity/exeobjparser_test.go

```diff
@@ -40,6 +40,10 @@ func TestExeObjParser(t *testing.T) {
 
 			key := fmt.Sprintf("%s_%s", builder, format)
 			t.Run(fmt.Sprintf("executableObject_%s_%s", format, builder), func(t *testing.T) {
+				if builder == "garble" && format == "pe" {
+					t.Skip("skipping test on garbled PE file: see https://github.com/elastic/beats/issues/35705")
```

Comment:
I'm not familiar with the testing framework, so maybe this question has an obvious answer:

If we're removing the test entirely, is it still necessary to mark it as skipped?

Reply:
We're iterating over [elf, macho, pe]×[go, garble]. The only way to remove a single element of a 3×2 is by an exception like this. We could just continue before the `Run` call, but I think leaving information about why we are doing this is useful.